coreprofile/10/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta EE Core Profile 10 (under development)"
+date: 2021-04-07
+summary: "Release of the Jakarta EE Core Profile 10"
+---
+The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
+
+The goals for this release are:
+1. To provide a profile that contains a set of Jakarta EE Specifications targeting smaller runtimes suitable for microservices as well as enabling compilation to native images.
+2. The TCK(s) for the Core profile should be possible to run independently of the Full Platform TCK.
```

Comment:
Does this statement imply that all proposed Specifications that make up the Core Profile also need to be run independently?  I think it should.  And, maybe we need to spell that out.

Reply:
I'm not sure this is the correct place to have debates on points like this but --- This would be a departure from previous releases. Currently, an implementation cannot claim platform compatibility (at any level) by simply passing the component TCKs. You can see this by noticing that the sum of the component TCK tests is always less than the sum of the Platform TCK tests -- Platform: 42,423 Web-Profile: 19,464 Stand alone: 18,261 (these don't include the additional TCKs that the platform specs call out -- cdi, di, bv, xml-binding). While this may be a discussion point, I don't know that it should be excluded in the initial prooposal. If indeed, this is indeed intended, I would be against starting out with the presumption that we would not have a core-profile specification TCK. If this it was later concluded -- after analysis -- this could be accomplished, that's fine, but I wouldn't want to go into this, with that presumption.

Reply:
Sorry, my wording was bad...  What I meant to say is that the specs that make up the core profile should be able to be run independently.  I understand that the Core Profile TCK may add some additional integration tests.  But, I thought one of the ideas going forward is to ensure that we have independent TCKs.

Reply:
I also think we should spell out the requirement:
All enclosed specs should have TCKs that can be run independently
The core profile integration TCK should run independently as well.

Reply:
As I re-read this statement and associated comments, I think the statement is sufficiently clear for a Plan Review.  The eventual goal is for all TCKs to have the ability to be executed independently -- whether we're talking an standalone component such as CDI or a profile such as Core or the full Platform.